include/boost/geometry/algorithms/area.hpp

```diff
@@ -216,15 +216,20 @@ struct area<MultiGeometry, multi_polygon_tag> : detail::multi_sum
 namespace resolve_strategy
 {
 
+template <typename Strategy>
 struct area
 {
-    template <typename Geometry, typename Strategy>
+    template <typename Geometry>
     static inline typename area_result<Geometry, Strategy>::type
     apply(Geometry const& geometry, Strategy const& strategy)
     {
         return dispatch::area<Geometry>::apply(geometry, strategy);
     }
+};
 
+template <>
+struct area<default_strategy>
```

Comment:
No, this was done from technical reasons. In order to instantiate `area_result` with correct arguments and because it's a return type of member function I had to do it this way. Otherwise I'd left it as it was.